crates/subspace-service/src/lib.rs

```diff
@@ -806,7 +843,7 @@ where
         }
     };
 
-    let import_queue_service = import_queue.service();
+    let import_queue_service = Arc::new(tokio::sync::Mutex::new(import_queue.service()));
```

Comment:
Do you just need import queue service in multiple places? If so you can call `import_queue.service()` more than once and get two service instances with handles to the same import queue. Or is there a different reason for this?

Reply:
You're still sharing it this way, what is the benefit of `Arc<Mutex<T>>` comparing to just `T` here? Seems like unnecessary complexity to me.

Reply:
Absolutely, it is like having a Arc and instead of accessing it from two places directly you wrapped it in additional `Arc<Mutex>>`. I simply do not understand why would you even do that.